Documentation/Home/GuidesAndTutorials.rst

```diff
@@ -58,10 +59,8 @@ section on this page.
    manuals to select a different version? The versions of the manual correspond to the TYPO3
    version. :ref:`Read more ... <usage-version-selector>`
 
-.. _tutguides-editors:
```

Comment:
I like the change but why are these anchors moved to the top?

Reply:
We try to preserve anchors in case they are being to linked to somewhere else. I think this is currently the case in the "Getting Started Tutorial". 

The anchors are always before a headline. What I did is change the headlines to normal text. So I moved them before the first headline on the page. The existing crosslinks (:ref:) will now still work and just link to the page but not the subsection.

Reply:
P.S. I did not render locally yet, so don't know how this looks.

Reply:
Thanks for the explanation.